movement/io/load_bboxes.py

```diff
@@ -180,6 +185,12 @@ def from_file(
         full video as the time origin. If False (default), the frame numbers
         in the VIA tracks .csv file are instead mapped to a 0-based sequence of
         consecutive integers.
+    frame_regexp : str, optional
```

Comment:
Maybe also mention that this argument is only relevant if `use_frame_numbers_from_file` is True.